src/pages/settings/Wallet/VerifyAccountPage.tsx

```diff
@@ -50,7 +55,7 @@ function VerifyAccountPage({route}: VerifyAccountPageProps) {
         if (!navigateBackTo) {
             return;
         }
-
+        Navigation.goBack();
         Navigation.navigate(navigateBackTo);
```

Comment:
@getusha This PR is ready for review. I think you might need to test the case I mentioned in our DM. 
Also another flow that used `VerifyAccountPage` and see if there's problem with this `goBack` here.

Reply:
There is a problem with both `dismissModal` and `goBack` for cases where `navigateBackTo` leads to RHP.

If you look closely at the video for web,  you'll see that the overlay doesn't work properly because we close the RHP and immediately open a new one.

To solve this, we can try to replace the validate account flow with the add bank account flow. This way RHP will stay opened the whole time.

Please try this `Navigation.navigate(navigateBackTo, CONST.NAVIGATION.TYPE.UP)`. This should force replace action. You don't need any `goBack()` and `dismissModal()` with this solution.

I think it should work with other cases e.g. replacing the whole RHP with central pane but it's something worth checking. 

Please let me know if the above works for you. Thanks! 🙇

Reply:
Thanks for your advice @adamgrzybowski. I'll give it a try 👍

Reply:
It's working well 🚀 . Should be ready for your final review @mountiny